src/sdk/namespace_fs.js

```diff
@@ -574,35 +580,51 @@ class NamespaceFS {
     }
 
     async complete_object_upload(params, object_sdk) {
+        let read_file;
+        let write_file;
         try {
             const { multiparts = [] } = params;
             multiparts.sort((a, b) => a.num - b.num);
             await this._load_multipart(params);
             const file_path = this._get_file_path(params);
             const upload_path = path.join(params.mpu_path, 'final');
-            const upload_stream = fs.createWriteStream(upload_path);
+            write_file = await nb_native().fs.open(DEFAULT_FS_CONFIG, upload_path, 'a');
```

Comment:
Why "a" ??? we do not want to keep appending to existing file in any case.
I would open as "w" or "wx".